### PR TITLE
Refactored Content component

### PR DESCRIPTION
**Pull request by atiqzaman-dsi**
1. Title prop for mobile view
2. Back button for mobile view

### CODE REVIEW

File: packages/client/src/components/interface/Header/Header.tsx [hunk 1]
```diff
@@ -527,12 +528,14 @@ class HeaderComp extends React.Component<IFullProps, IState> {
     )
 
     return (
-      <AppHeader
-        id="register_app_header"
-        desktopRightMenu={rightMenu}
-        title={title}
-        {...mobileHeaderActionProps}
-      />
+      <div {...headerProps}>
```
Comment (rikukissa) [inline packages/client/src/components/interface/Header/Header.tsx:531]: image

This breaks the top header's sticky positioning. Can we expand the props to AppHeader? Notice also that `this.props`has many non-html attributes that will end up rendered in the DOM, which causes console warnings. Can we either omit all non-html props from the object before expanding OR only explicitly take the ones we want to add to the element?

Comment (atiqzaman-dsi) [reply packages/client/src/components/interface/Header/Header.tsx:531]: I got you @rikukissa

Comment (atiqzaman-dsi) [reply packages/client/src/components/interface/Header/Header.tsx:531]: @rikukissa I was thinking to remove the DOM props Header and handle show/hide in RecordAudit using width state based on screen size.
`{width > grid.breakpoints.lg && <Header />}`
What do you think?

Comment (rikukissa) [reply packages/client/src/components/interface/Header/Header.tsx:531]: Can you do it with CSS media queries? I think if yes, then that's the best approach 👍

**Review by rikukissa — APPROVED** [review]

**Pull request merged**